### DISCUSSION TITLE
Plans to enable Docker builds from Master branch again?

### DISCUSSION THREAD

**Discussion by Jdiesel87** [advplyr/audiobookshelf#442] [General]
It's been over a month since the last Docker build was pushed, is there any plans to push more frequent builds from the Master branch? Maybe under a Dev or Nightly tag?

**Comment by advplyr** [comment]
Right now the server won't work with the mobile apps. The server has gotten a major rewrite and now the mobile apps are getting worked on to catch up.
Once v2 is released we'll be setting up auto-builds and a more standard release cycle.

**Reply by Jdiesel87** [reply]
Thanks for the update!
